PluginsAndFeatures/azure-toolkit-for-intellij/resources/whatsnew/whatsnew.md

```diff
@@ -1,6 +1,27 @@
 <!-- Version: 3.51.0 -->
 # What's new in Azure Toolkit for IntelliJ
 
+## 3.55.0
+### Added
+- New azure resource connector explorer for connection management
+- Add support for IntelliJ proxy
```

Comment:
the sentence since it is our task name, it is much difficult to understand the meaning, add snapshot of proxy setting too(socks not http)